From c543e4e63e50198306e9499a76c13fd0961e2dca Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <josef.schlehofer@nic.cz>
Date: Thu, 24 Dec 2020 20:52:29 +0100
Subject: [PATCH] ruby-breakpoint: remove this package

Sometimes, this package is not compiled. This decreases jobs on our
build servers and then it works.

This package it is dependency for Foris,
which is not present in HBD branch (current snapshots of OpenWrt
20.xx/21.xx). So we can drop it without replacement.
---
 lang/ruby-breakpoint/Makefile | 28 ----------------------------
 1 file changed, 28 deletions(-)
 delete mode 100644 lang/ruby-breakpoint/Makefile

diff --git a/lang/ruby-breakpoint/Makefile b/lang/ruby-breakpoint/Makefile
deleted file mode 100644
index 37697b107..000000000
--- a/lang/ruby-breakpoint/Makefile
+++ /dev/null
@@ -1,28 +0,0 @@
-#
-## Copyright (C) 2019 CZ.NIC z.s.p.o. (http://www.nic.cz/)
-#
-## This is free software, licensed under the GNU General Public License v2.
-# See /LICENSE for more information.
-# #
-#
-include $(TOPDIR)/rules.mk
-
-PKG_NAME:=ruby-breakpoint
-PKG_VERSION:=2.7.1
-PKG_RELEASE:=1
-PKG_MAINTAINER:=CZ.NIC <packaging@turris.cz>
-
-HOST_BUILD_DEPENDS:=ruby/host
-
-include $(INCLUDE_DIR)/host-build.mk
-include $(INCLUDE_DIR)/package.mk
-
-Build/Compile:=:
-Host/Compile:=:
-
-define Host/Install
-	$(STAGING_DIR_HOSTPKG)/bin/gem install breakpoint -v $(PKG_VERSION)
-endef
-
-$(eval $(call HostBuild))
-$(eval $(call Build/DefaultTargets))
-- 
2.25.1

